app/controllers/transcribe_controller.rb

```diff
@@ -312,6 +312,37 @@ def still_editing
     render nothing: true
   end
 
+  def goto_next_untranscribed_page
+    
+    next_page_path = landing_page_path
+    flash[:notice] = "There are no more pages to transcribe in this collection! Check out some of our other projects."
```

Comment:
I wonder if we should redirect them to the project owner's page instead?

Reply:
I agree.  Sending them to the owner's page would keep them closer to the organization.

Reply:
That makes sense.